lib/jazzy/executable.rb

```diff
@@ -29,11 +29,10 @@ def execute_command(executable, args, raise_on_failure, env: {})
         options = { stdout: stdout, stderr: stderr, status: true }
         status  = Open4.spawn(env, bin, *args, options)
         unless status.success?
-          full_command = "#{bin.shellescape} #{args.map(&:shellescape)}"
-          output = stdout.to_s << stderr.to_s
-          if raise_on_failure
-            raise "#{full_command}\n\n#{output}"
+          if exit_on_failure
+            exit status.exitstatus
```

Comment:
This will break cocoadocs. I'd prefer to raise a specific exception subclass that the bin rescues and prints out